TumbleBitSetup.Tests/ProtocolTests.cs

```diff
@@ -108,18 +159,7 @@ public void test_I2OSP_2()
         }
 
         [TestMethod()]
-        public void Test3E_multiple()
-        {
-            /*
-             * Repeat test 3E a 100 times with a different modulus N each time.
-             * (This assumes that TestUtils.GenQ() will give a new q every time)
-            */
-            for (int i = 0; i < 100; i++)
-                Test3E();
-        }
-
-        [TestMethod()]
-        public void shortN()
+        public void ShortN()
```

Comment:
Done, check f614f1c7b939e13b41070d36c54ea7d0de25fb42
Also note that in the case of 500 bits to the prover in poupard-stern the 2^{|N|-1}/(N-phi)2^k check is triggered and the proving stops. Is that expected to happen? is that what this check is preventing?